promoted_python_delivery_client/model/execution_server.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class ExecutionServer(Enum):
+    UNKNOWN = 0
```

Comment:
Will the enum serialize the number or the string name?  I assume number since the actual unknown name is UNKNOWN_EXECUTION_SERVER.